planetary-variables/crop-biomass/script.js

```diff
@@ -0,0 +1,35 @@
+//VERSION=3
+function setup() {
+  return {
+    input: ["CB", "dataMask"],
+    output: { bands: 4, sampleTYPE: "AUTO" },
```

Comment:
output: { bands: 4, sampleType: "FLOAT32" },

Reply:
"AUTO" is fine for visualization. It returns 0-1 scaled to UINT8. Basically AUTO converts reflectances to RGB